ops/_private/harness.py

```diff
@@ -3176,6 +3176,26 @@ def autostart_services(self, timeout: float = 30.0, delay: float = 0.1):
                 self._service_status[name] = pebble.ServiceStatus.ACTIVE
 
     def replan_services(self, timeout: float = 30.0, delay: float = 0.1):
+        for name, check in self._render_checks().items():
+            if check.startup != pebble.CheckStartup.DISABLED:
```

Comment:
Is this for Scenario or only Harness?

Reply:
Both, Scenario re-uses most of the mock Pebble client.

Note that this is still draft, there's a bunch of stuff I haven't pushed yet, particularly around testing.